### PR TITLE
fix(staging/tidb): fix curl download unstable

### PR DESCRIPTION
**Pull request by purelind**
To avoid abnormal interruptions during download, use `wget` instead of `curl` to download file from fileserver.

### CODE REVIEW

File: jobs/pingcap/tidb/latest/merged_integration_br_test.groovy [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // REF: https://<your-jenkins-server>/plugin/job-dsl/api-viewer/index.html
 pipelineJob('pingcap/tidb/merged_integration_br_test') {
+    disabled(true)
```
Comment (wuhuizuo) [inline jobs/pingcap/tidb/latest/merged_integration_br_test.groovy:3]: How about controlling by prow bot to:
- report or not report git commit status
- run on not run

File: pipelines/pingcap/tidb/latest/merged_integration_br_test.groovy [hunk 2]
```diff
@@ -76,6 +76,12 @@ pipeline {
                     cache(path: "./bin", filter: '**/*', key: "ws/${BUILD_TAG}/br") {
                         sh label: 'tidb-server', script: 'ls bin/tidb-server || make'
                         sh label: 'build-for-br-integration-test', script: 'make build_for_br_integration_test'
+                        sh label: "download dependency", script: """
+                        chmod +x ../scripts/pingcap/br/*.sh
+                        ${WORKSPACE}/scripts/pingcap/br/integration_test_download_dependency.sh master master master master master http://fileserver.pingcap.net
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/latest/merged_integration_br_test.groovy:81]: less readability, consider to define shell vars, or make the shell script better with named args.

File: pipelines/pingcap/tidb/latest/merged_integration_br_test.groovy [hunk 3]
```diff
@@ -112,18 +118,18 @@ pipeline {
...
                             dir('tidb') {
                                 cache(path: "./", filter: '**/*', key: "git/pingcap/tidb/rev-${GIT_MERGE_COMMIT}") { 
                                     sh """git status && ls -alh""" 
                                     cache(path: "./bin", filter: '**/*', key: "ws/${BUILD_TAG}/br") {
-                                        sh label: 'tidb-server', script: 'ls bin/tidb-server && chmod +x bin/tidb-server'
-                                        sh 'chmod +x ../scripts/pingcap/br/*.sh'
-                                        sh "${WORKSPACE}/scripts/pingcap/br/integration_test_download_dependency.sh master master master master master http://fileserver.pingcap.net"
+                                        sh label: 'tidb-server version', script: 'ls bin/tidb-server && chmod +x bin/tidb-server && ./bin/tidb-server -V'  
+                                        sh label: 'tikv-server version', script: 'ls bin/tikv-server && chmod +x bin/tikv-server && ./bin/tikv-server -V'
+                                        sh label: 'pd-server version', script: 'ls bin/pd-server && chmod +x bin/pd-server && ./bin/pd-server -V'
```
Comment (wuhuizuo) [inline pipelines/pingcap/tidb/latest/merged_integration_br_test.groovy:132]: consider to merge to single shell step to save Jenkins performance.

File: jobs/pingcap/tidb/latest/merged_integration_br_test.groovy (hunk 1 above)
Comment (purelind) [reply jobs/pingcap/tidb/latest/merged_integration_br_test.groovy:3]: pipeline test found unstable, need to debug, prow is not available for takeover at this time

**Pull request merged**